sqrl-planner/src/test/java/com/datasqrl/config/CompilerConfigImplTest.java

```diff
@@ -50,7 +50,19 @@ void givenCostModel_whenInvalid_thenThrowsError() {
 
     assertThatThrownBy(compilerConfig::getCostModel)
         .isInstanceOf(CollectedException.class)
-        .hasMessage(
-            "Value [asd] for key [cost-model] is not valid. Must be one of [DEFAULT, READ, WRITE]");
+        .hasMessageContaining(
+            "String \"asd\": not one of the values accepted for Enum class: [DEFAULT, READ, WRITE]");
+  }
+
+  @Test
+  void givenCostModel_whenLowercase_thenThrowsError() {
+    config.setProperty("cost-model", "read");
```

Comment:
sqrl is extremely case insensitive in most cases.

We even go out of our way to locate files ignoring case on case sensitive file systems